### ISSUE TITLE
migrate to AndroidX

### ISSUE THREAD

**Issue by The-Redhat** [droibit/flutter_custom_tabs#9]

**Comment by Gurkengewuerz** [comment]
@The-Redhat already requested a PR 2 days ago but since now no reaction from @droibit.
You can test the AndroidX Support by using my update.
```
dependencies:
  flutter_custom_tabs:
    git:
      url: https://github.com/Gurkengewuerz/flutter_custom_tabs.git
      ref: workaround
```

**Comment by The-Redhat** [comment]
Thank you for the PR. I hope it gets merged soon.

**Comment by baoxiehao** [comment]
> @The-Redhat already requested a PR 2 days ago but since now no reaction from @droibit.
> You can test the AndroidX Support by using my update.
> 
> ```
> dependencies:
>   flutter_custom_tabs:
>     git:
>       url: https://github.com/Gurkengewuerz/flutter_custom_tabs.git
>       ref: workaround
> ```

Happy to see the workaround and it's working now.
But my problem is that the first launch will be quite slow as to webview, but later it is quite fast and the transition animation is also smoother.
Great!

**Comment by The-Redhat** [comment]
@droibit can you please merge the mentioned pr above.

**Comment by baoxiehao** [comment]
> Unable to load Maven meta-data from https://jitpack.io/com/github/Gurkengewuerz/CustomTabsLauncher/launcher/androidx-SNAPSHOT/maven-metadata.xml.
            > Could not HEAD 'https://jitpack.io/com/github/Gurkengewuerz/CustomTabsLauncher/launcher/androidx-SNAPSHOT/maven-metadata.xml'.

**Comment by Gurkengewuerz** [comment]
Strange, i can recompile the project without a problem. Can you provide the full compile log as a gist please?

**Comment by baoxiehao** [comment]
I've tried the VPN and it doesn't work also. Strange. Few days ago it worked always.

```
  flutter_custom_tabs:
    git:
      url: https://github.com/Gurkengewuerz/flutter_custom_tabs.git
      ref: workaround
```

```
Initializing gradle...
Resolving dependencies...
Gradle task 'assembleDebug'...
         *********************************************************
WARNING: This version of local_auth will break your Android build if it or its dependencies aren't compatible with AndroidX.
         See https://goo.gl/CP92wY for more information on the problem and how to fix it.
         This warning prints for all Android build failures. The real root cause of the error may be unrelated.
         *********************************************************

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:debugCompileClasspath'.
   > Could not resolve com.github.Gurkengewuerz.CustomTabsLauncher:launcher:androidx-SNAPSHOT.
     Required by:
         project :app > project :flutter_custom_tabs
      > Could not resolve com.github.Gurkengewuerz.CustomTabsLauncher:launcher:androidx-SNAPSHOT.
         > Unable to load Maven meta-data from https://jitpack.io/com/github/Gurkengewuerz/CustomTabsLauncher/launcher/androidx-SNAPSHOT/maven-metadata.xml.
            > Could not HEAD 'https://jitpack.io/com/github/Gurkengewuerz/CustomTabsLauncher/launcher/androidx-SNAPSHOT/maven-metadata.xml'.
               > Connect to jitpack.io:443 [jitpack.io/104.24.23.62, jitpack.io/104.24.22.62, jitpack.io/2606:4700:20:0:0:0:6818:173e, jitpack.io/2606:4700:20:0:0:0:6818:163e] failed: No route to host (connect failed)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 27s
Finished with error: Gradle task assembleDebug failed with exit code 1
```

**Comment by Gurkengewuerz** [comment]
> Thank you for the PR. I hope it get's merged soon.

First step done. The PR in the native plugin is merged into the develop branch. Now we need the flutter plugin :)

**Comment by droibit** [comment]
Thank you for waiting 🙇    
I just published the latest v0.5.0 of this package to pub.

Thanks!!

**Comment by baoxiehao** [comment]
My problem should be a network problem cause I tried it yesterday in my home and it compiled successfully. Thanks. Hope to see the 0.5.0 soon.
